apps/frontend/src/components/ui/AdminApplicationPeriodCard.tsx

```diff
@@ -52,6 +59,31 @@ export default function AdminApplicationPeriodCard({ period }: { period: Applica
     await mutate(`/application-periods/${period.id}`);
   };
 
+  const onDummyExport = async () => {
+    const now = new Date();
+    const blob = await pdf(
+      <PassExport
+        mock
+        periodId={period.id}
+        applicationData={[mockApplication]}
+        periodName={`${now.getFullYear()}. ${now.getMonth() < 7 ? 'tavasz' : 'ősz'}`}
+      />
+    ).toBlob();
+    // eslint-disable-next-line no-undef
+    const a = document.createElement('a');
+    a.style.display = 'none';
+    // eslint-disable-next-line no-undef
+    document.body.appendChild(a);
+
+    // eslint-disable-next-line no-undef
+    const url = window.URL.createObjectURL(blob);
+    a.href = url;
+    a.download = `schbody_pass_mock_export_${Date.now()}.pdf`;
+    a.click();
+    // eslint-disable-next-line no-undef
+    window.URL.revokeObjectURL(url);
+  };
```

Comment:
this can be extracted to another function, the same is done in #86, so if that's merged I'll update this